kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/MarshallingFormat.java

```diff
@@ -57,19 +58,24 @@ public static MarshallingFormat fromId(int id) {
         }
     }
 
+    
     public static boolean isStrictType(String type) {
+        String strictParam  = (String)buildParameters(type).get(Marshaller.MARSHALLER_PARAMETER_STRICT);
+        return strictParam != null && Boolean.parseBoolean(strictParam);
+    }
+    
+    public static Map<String,Object> buildParameters(String contentType)
+    {
         int idx;
-        if ((idx = type.indexOf(';')) < 0 || (idx + 1) == type.length()) {
-            return false;
-        }
-        // we map parameter=1, pararmeter=2,.... into a map
-        Map<String, String> parameters = Arrays.asList(type.substring(idx + 1).split(","))
-                                               .stream()
-                                               .filter(e -> e.split("=").length > 1) // remove bad parameters
-                                               .map(e -> e.split("="))
-                                               .collect(Collectors.toMap(e -> ((String[]) e)[0].trim(), e -> ((String[]) e)[1]));
-        return Boolean.parseBoolean(parameters.get(Marshaller.MARSHALLER_PARAMETER_STRICT));
+        return ((idx = contentType.indexOf(';')) < 0 || (idx + 1) == contentType.length()) 
+                ? Collections.emptyMap() 
+                : Arrays.stream(contentType.substring(idx + 1).split(",")).
```

Comment:
@gmunozfe how was your trick to make the split(",") more robust ? split("\\s*,\\s*) ?

Reply:
, and ;  are not a valid character in content type, this proposed changed will not make the code more robust, but slower

Reply:
`trim().split("\\s*,\\s*")`
trim() method removes leading and trailing spaces in the input string, and the regex itself handles the extra spaces around delimiter.